global/apps/homepage/views.py

```diff
@@ -146,11 +146,77 @@ def _get_reading_list(country_code):
 
 @cache_control(no_cache=True)
 def about(request):
+    country_code = request.META.get('HTTP_CF_IPCOUNTRY')
+    if country_code is None:
+        country_code = request.META.get(
+            'GEOIP_COUNTRY_CODE',
+            '--',
+        )
+
+    # Fetch the list of contributors from the various missions. Use a
+    # set() because we only want to list each contributor once.
+    contributors = set()
+    for mission in list(Mission.Query(request.redis_conn)):
+        if not mission.incomplete:
+            contributors = contributors | set(mission.copy.get('cleaners', []))
+    # Some people have contributed code changes but haven't worked on
+    # missions.
+    contributors = contributors | {
+        'Adam Johnson',
+        'Tom Morris',
+        'Christopher Stumm',
+    }
+
+    # Strip out any of the original team, because they're already
+    # listed explicitly.
+    contributors = contributors - {
+        'Matthew Ogle',
+        'Russ Garrett',
+        'Hannah Donovan',
+        'Chris Govias',
+        'Gavin O\'Carroll',
+        'Ryan Alexander',
+        'James Aylett',
+        'George Brocklehurst',
+        'David Brownlee',
+        'Ben Firshman',
+        'Mark Norman Francis',
+        'Andrew Godwin',
+        'Steve Marshall',
```

Comment:
We've now got the original team listed in the view and in the template. Is it worth passing our names to the template too? Maybe not, since it's never going to change.